app/controllers/public.js

```diff
@@ -11,6 +11,11 @@ export default Controller.extend({
   displayEndDate: computed('model.startsAtDate', 'model.endsAtDate', function() {
     return !moment(this.model.startsAtDate).isSame(this.model.endsAtDate, 'minute');
   }),
+  displaySideMenu: computed('session.currentRouteName', function() {
+    if (this.get('session.currentRouteName')) {
```

Comment:
Dont use ES5 getter.

Reply:
then what ??

Reply:
should i migrate it to ES6?

Reply:
No, ~this.get('session')~ this.session

Reply:
in the same file :
```
smallLead: computed('session.currentRouteName', function() {
    if (this.get('session.currentRouteName')) {
      return this.get('session.currentRouteName') !== 'public.index';
    }
  }),
```
Should i change  ES5 here too ?

Reply:
in the same file :

```
smallLead: computed('session.currentRouteName', function() {
    if (this.get('session.currentRouteName')) {
      return this.get('session.currentRouteName') !== 'public.index';
    }
  }),
```
Should i change ES5 here too ?